evap/student/views.py

```diff
@@ -310,15 +371,15 @@ def vote(request, evaluation_id):  # noqa: PLR0912
         RatingAnswerCounter.objects.filter(contribution__evaluation=evaluation).update(id=F("id"))
         TextAnswer.objects.filter(contribution__evaluation=evaluation).update(id=F("id"))
 
+        # TODO@Felix: do this in on_commit? (or via select for update?)
         if not evaluation.can_publish_text_results:
             # enable text result publishing if first user confirmed that publishing is okay or second user voted
             if (
                 request.POST.get("text_results_publish_confirmation_top") == "on"
                 or request.POST.get("text_results_publish_confirmation_bottom") == "on"
-                or evaluation.voters.count() >= 2
+                or voters.filter(pk=evaluation.pk).count() >= 2
```

Comment:
Does this new query still include the new voter, or is it constrained to the old set of voters? I would think that it is a fresh query, which then means that we can still use `evaluation.voters` right?

Currently, I am a bit confused why the `voters` from before is used, so a change or an explanatory comment would be good :)